### DISCUSSION TITLE
Removing the steamid check upon being placed as a dupe

### DISCUSSION THREAD

**Discussion by Cheesecakd** [wiremod/wire#2459] [Suggestions]
Problem:
While its meant to protect e2 creators from having their e2 duped without permission, the steamid check when placing a dupe does not achieve its desired effect. 
SInce the WireMod workshop update, workshop dupe users are reporting to me that the e2 contraptions are placed with an error that forces them to read the code using the expression 2 editor and remove the #error in line 1.
The problem is not everyone who uses dupe want to learn how to pull out expression 2 tool from wire and remove line 1 in whatever e2 their dupe contains.

Screenshots:
![18367802 11_image](https://user-images.githubusercontent.com/116293115/197012534-4ce3036d-55a9-4e76-b177-ffd5b025d4e5.png)
![18404388 51_image](https://user-images.githubusercontent.com/116293115/197012744-2ae7337f-7a29-4b1a-8169-d06abfd2d32e.png)

Solution: 
Remove the code / add options to protect unfinished e2s from being duped

**Comment by Derpius** [comment]
It’s not meant to protect E2 users from having their code duped, it’s to protect the 5 year old admins on servers from spawning pub dupes that concmd everyone to admin, and similar shit that doesn’t require the user to be an admin

**Comment by Cheesecakd** [comment]
Then maybe it’s more effective to cross out concmd on dupe/ #error on concmd()?

**Comment by Cheesecakd** [comment]
Or just turn off concmd when duping e2s

**Comment by Derpius** [comment]
Concmd isn’t the only function that could cause problems

Also this has already been discussed in discord with potential solutions put forward. The compromise solution that was chosen (not the best but the easiest to implement) was adding back in the friendwrite convar for dupe protection

I barely have time to work on my own projects now I’m a full time software developer, never mind wiremod, so you’re gonna have to wait for someone else to implement that though

**Reply by Cheesecakd** [reply]
Hmm ok. I guess that’s WireMod
